pkg/block/block_linux.go

```diff
@@ -410,15 +410,15 @@ func partitionInfo(paths *linuxpath.Paths, part string) (string, string, bool) {
 	return "", "", true
 }
 
-type mtabEntry struct {
+type mountInfoEntry struct {
```

Comment:
we could probably just call this `mountEntry`?

Reply:
sure thing